### ISSUE TITLE
1.25.0 broken packages

### ISSUE THREAD

**Issue by theofficialgman** [minbrowser/min#2023]
- **Min Version:** 1.25.0
- **Operating system:** linux arm64

## Expected Behavior

packages are packaged correctly

## Actual Behavior

```
Note, selecting 'min:aarch64' instead of './min-1.25.0-arm64.deb'
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help resolve the situation:

The following packages have unmet dependencies:
 min:aarch64 : Depends: libsecret-1-dev:aarch64 but it is not installable
               Depends: libasound2:aarch64 but it is not installable
               Depends: libc6:aarch64 but it is not installable
               Depends: libcap2:aarch64 but it is not installable
               Depends: libgtk2.0-0:aarch64 but it is not installable
               Depends: libudev0:aarch64 but it is not installable or
                        libudev1:aarch64 but it is not installable
               Depends: libgcrypt11:aarch64 but it is not installable or
                        libgcrypt20:aarch64 but it is not installable
               Depends: libnotify4:aarch64 but it is not installable
               Depends: libnss3:aarch64 but it is not installable
               Depends: libxss1:aarch64 but it is not installable
               Depends: libxtst6:aarch64 but it is not installable
               Depends: python:aarch64 but it is not installable or
                        python3:aarch64 but it is not installable
               Recommends: libappindicator3-1:aarch64 but it is not installable
E: Unable to correct problems, you have held broken packages.
```
aarch64 does NOT exist in debian. the official architectures are `arm64` and `armhf`. please correct your debs to target these architectures

## To Reproduce

install .deb from 1.25.0 release

**Comment by PalmerAL** [comment]
CC @cog1to 

I'm wondering if this will fix it: https://github.com/minbrowser/min/commit/2fbf27104cd41845947ef54ace3d8b40fadd9439 . Will post a build soon to test.

I'm also seeing an issue with the x64 deb build: after installing, it doesn't seem to be launchable from the command line. With older versions, launching with just `min` works; the new build that installs in `/opt/Min/min` doesn't (`which min` finds nothing).

**Comment by cog1to** [comment]
thanks for notifying. I think I tested the deb package on Ubuntu, I'm surprised it seems broken on Debian. I hope your change fixes it. If not, I'll try to experiment with it as well.

as for the new install path: yes, I mentioned that in my initial PR. probably can be fixed by adding a `ln -s /opt/Min/min /usr/bin/min` or something like that to the `resources/postinst_script` script.

**Comment by PalmerAL** [comment]
Looking back at the PR, I don't think either of us tested the arm build, just the x86 one. (and I don't have any hardware to test this on still).

Adding a symlink sounds like a reasonable solution.

Also, `/opt/Min` should be lowercase I think? Unless there's a standard that applications there are usually uppercase (I'm not sure).

**Comment by theofficialgman** [comment]
yes you can already tell from the correctly generated `control` files that that particular issue is fixed
```
Package: min
Version: 1.25.0
License: Apache-2.0
Vendor: Min Developers <280953907a@zoho.com>
Architecture: arm64
Maintainer: Min Developers <280953907a@zoho.com>
Installed-Size: 220233
Depends: libsecret-1-dev, gconf2, gconf-service, libasound2, libc6, libcap2, libgtk2.0-0, libudev0 | libudev1, libgcrypt11 | libgcrypt20, libnotify4, libnss3, libxss1, libxtst6, python | python3, xdg-utils
Recommends: libappindicator3-1
Section: Network
Priority: optional
Homepage: https://github.com/minbrowser/min#readme
Description: A web browser with smarter search, improved tab management, and built-in ad blocking. Includes full-text history search, instant answers from DuckDuckGo, the ability to split tabs into groups, and more.
  Min is a fast, minimal browser that protects your privacy.
```

however, this app is still broken. there are constant javascript errors:
![Screenshot from 2022-06-23 18-11-10](https://user-images.githubusercontent.com/28281419/175423842-4ecbc6f0-9a2a-46ba-b673-e39dc8a683d6.png)

**Comment by PalmerAL** [comment]
OK great, thanks for checking. That issue will be fixed also in the next build (see #2022). Sorry for all the issues.

**Comment by theofficialgman** [comment]
can confirm it works all now. closing
